cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -922,24 +907,60 @@ function Configure-GcePdTools {
     New-Item -path $PsHome\profile.ps1 -type file
   }
 
-    Add-Content $PsHome\profile.ps1 `
+  Add-Content $PsHome\profile.ps1 `
```

Comment:
nit: also intent 3 lines below?

Reply:
Indenting the lines below would indent the contents of the file, which we don't want.